Custom-Video-Driver/Lets-Build-OTPublisher/ViewController.m

```diff
@@ -17,6 +17,7 @@
 // iOS supports multi cam only on higher end devices (e.g, >= A12 CPU)
 // The capturer will return nil if you try to run multi cam samples on an unsupported device!
 #define USE_MULTICAM_SESSION 0
+#define TAP_TO_UNPUBLISHPUBLISH_ 1
```

Comment:
Let us not have this file, at all , in this  PR. 
Let the user decide what flow they want to test/use. We just give them the fixed Capturere